.travis.yml

```diff
@@ -69,6 +69,8 @@ before_script:
 script:
   - phpdbg -qrr vendor/bin/phpunit --coverage-text --coverage-clover build/logs/clover.xml
   - PHP_CS_FIXER_IGNORE_ENV=1 php vendor/bin/php-cs-fixer --diff --dry-run -v fix
+  - chmod +x benchmarks/bin/php_no_xdebug
+  - PATH=$PATH:$(pwd)/benchmarks/bin vendor/bin/phpbench run --report=aggregate
```

Comment:
Not sure we have to run that for each build job. It's probably better to have a matrix that separates builds which run (unit) tests and then only one job (one PHP version is enough) to run these benchmarks.

Reply:
Fixed. Fully agree :)